### PR TITLE
Bump juju/utils dependency w/ fixes.

### PR DESCRIPTION
**Pull request by Veebers**
Bumped juju/utils version in juju/juju which broke charmstore.v5-unstable due to API changes.

Update charmstore.v5-unstable in regards to those API changes (as well as bump the juju/utils version to match too).

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2168/

**Comment by Veebers** [comment]
I'm not sure why this is failing in CI, seems it's not pulling the deps for juju/utils? The tests pass for me when I run them locally (in a lxd container) using the Makefile provided commands.

**Comment by mhilton** [comment]
It looks like you're missing a dependency on gopkg.in/juju/httprequest.v1 `make create-deps` should sort that out for you.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2169/

**Comment by Veebers** [comment]
@mhilton Ah I see, thanks for clarifying for me :-)

File: dependencies.tsv [hunk 1]
```diff
@@ -33,6 +33,7 @@ golang.org/x/sys	git	7a6e5648d140666db5d920909e082ca00a87ba2c	2017-02-01T05:12:4
 gopkg.in/check.v1	git	4f90aeace3a26ad7021961c297b22c42160c7b25	2016-01-05T16:49:36Z
 gopkg.in/errgo.v1	git	442357a80af5c6bf9b6d51ae791a39c3421004f3	2016-12-22T12:58:16Z
 gopkg.in/goose.v2	git	455416024500800de8dadbcacfa22681b6bb818d	2017-09-04T09:17:38Z
+gopkg.in/httprequest.v1	git	35158f716c228fdf58b5a5b80cf331302d10160a	2017-11-03T09:19:05Z
```
Comment (mhilton) [inline dependencies.tsv:36]: This unfortunately means there are two versions of this package linked in the code (the other being github.com/juju/httprequest). I've checked and the use of that package in the charmstore code itself is fairly limited. Could you please update the instances of the old package with the new one?

**Review by mhilton — CHANGES_REQUESTED** [review]
The code that's there is fine but if you could just do a little housekeeping that would be great.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2170/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/charmstore/2171/

**Comment by Veebers** [comment]
OK got it sorted.

I had to take into account the change from httprequest.ErrorMapper as well as update the expected strings in the test (due to the difference betweeen juju/httprequest and httprequest.v1 error message format).

**Review by mhilton — APPROVED** [review]
LGTM

**Comment by rogpeppe** [comment]
LGTM

**Comment by rogpeppe** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/charmstore-merge

**Pull request merged**